packages/lavamoat-node/src/kernel.js

```diff
@@ -55,7 +55,7 @@ function createKernel({
     getRelativeModuleId,
     prepareModuleInitializerArgs,
     getExternalCompartment,
-    globalThisRefs: ['global', 'globalThis'],
+    globalThisRefs: ['window', 'self', 'global', 'globalThis', 'top', 'frames', 'parent'],
```

Comment:
(must) Do not introduce new globals in node. None of these exist or are supposed to.

Reply:
9258717